actix-web-httpauth/src/headers/www_authenticate/challenge/mod.rs

```diff
@@ -10,3 +10,27 @@ pub trait Challenge: TryIntoHeaderValue + Debug + Display + Clone + Send + Sync
     /// Converts the challenge into a bytes suitable for HTTP transmission.
     fn to_bytes(&self) -> Bytes;
 }
+
+/// This is particularly useful for writing constructs such as
+/// `AuthenticationError::new("Authentication required")`
+impl Challenge for &'static str {
+    fn to_bytes(&self) -> Bytes {
+        (*self).into()
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use test_strategy::proptest;
+
+    use super::*;
+
+    #[proptest]
+    fn roundtrip_static_str(input: Box<str>) {
```

Comment:
I'm not sure if that conversion really needs to use such expensive testing, isn't it enough to add a basic test? We should be cautious to add an additional test framework just for one test.